falcon/testing/helpers.py

```diff
@@ -316,6 +316,33 @@ def invoke_coroutine_sync(coroutine, *args, **kwargs):
     )
 
 
+def runs_sync(coroutine):
```

Comment:
Should this reside in `falcon/util/sync.py` instead?

Reply:
Good question. It is a thin wrapper around `invoke_coroutine_sync`, and `invoke_coroutine_sync` is actually defined here. Maybe we should move both then?

Reply:
After thinking about this some more, I do like the idea of just colocating all the sync/async adapters in util/sync, even if some of them are really only intended for use in tests. And we may be able to make some of these adapters more efficient later on using greenletio.

Reply:
That makes sense. I'll move both into `falcon/util/sync.py`.

Reply:
Sed and done.